statchatui/src/Home/Home.jsx

```diff
@@ -27,22 +43,29 @@ const Home = ({ onStatsChange, stats }) => {
       console.error("Error updating profile:", error);
     }
 
-    const gamertag = formData.gamertag;
-    const platform = formData.platform;
     const apiKey = process.env.REACT_APP_API_KEY;
```

Comment:
Since you have multiple APIs now, it might be good to give this a more specific name, like `APEX_API_KEY`